src/renderer/bootstrap.tsx

```diff
@@ -19,6 +20,17 @@ import { App } from "./components/app";
 import { LensApp } from "./lens-app";
 import { themeStore } from "./theme.store";
 
+/**
+ * If this is a development buid, wait a second to attach
+ * Chrome Debugger to renderer process
+ * https://stackoverflow.com/questions/52844870/debugging-electron-renderer-process-with-vscode
+ */
+async function attachChromeDebugger() {
+  if (remote.process.defaultApp) {
```

Comment:
Is this for checking if the app in development mode? (correct me if I am wrong) If so, there is `isDevelopment` from `./src/common/vars`, would like be be better?

Reply:
Wasn't aware of this one, thanks!